### ISSUE TITLE
Kubeflow TFX Taxi Pipeline BigQuery missing project information

### ISSUE THREAD

**Issue by gilbblig** [tensorflow/tfx#363]
I followed instructions of the [Jupyter Notebook](https://github.com/kubeflow/pipelines/blob/master/samples/tfx-oss/TFX%20Example.ipynb) file with minor changes (e.g. TFX 0.12 didn't yield a pipeline for me, so I used TFX 0.13) to create a pipeline. The Notebook calls [this script](https://github.com/tensorflow/tfx/blob/master/tfx/examples/chicago_taxi_pipeline/taxi_pipeline_kubeflow.py), which I adjusted accordingly.

Querying BQ in the first step then causes the following error: 

`INFO:tensorflow:Generating examples.
INFO:tensorflow:Generating examples.
INFO:root:Missing pipeline option (runner). Executing pipeline using the default runner: DirectRunner.
INFO:root:Running pipeline with DirectRunner.
ERROR:root:Exception at bundle <apache_beam.runners.direct.bundle_factory._Bundle object at 0x7f06a548e4d0>, due to an exception.
 Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/apache_beam/runners/direct/executor.py", line 343, in call
    finish_state)
  File "/usr/local/lib/python2.7/dist-packages/apache_beam/runners/direct/executor.py", line 383, in attempt_call
    result = evaluator.finish_bundle()
  File "/usr/local/lib/python2.7/dist-packages/apache_beam/runners/direct/transform_evaluator.py", line 318, in finish_bundle
    with self._source.reader() as reader:
  File "/usr/local/lib/python2.7/dist-packages/apache_beam/io/gcp/bigquery.py", line 382, in reader
    kms_key=self.kms_key)
  File "/usr/local/lib/python2.7/dist-packages/apache_beam/io/gcp/bigquery_tools.py", line 835, in __init__
    'Missing executing project information. Please use the --project '
RuntimeError: Missing executing project information. Please use the --project command line option to specify it.`

Please tell if additional logging or further information on my setup is required. For now I was able to circumvent this problem via taking examples from a .csv.

**Comment by htahir1** [comment]
@gilbblig I think the problem is that there is a small typo in the taxi_pipeline_kubeflow.py example in line 105 here
```
# Beam args to run data processing on DataflowRunner.
_beam_pipeline_args = [
    '--runner=DataflowRunner',
    '--experiments=shuffle_mode=auto',
    '--project=' + _project_id,
    '--temp_location=' + os.path.join(_output_bucket, 'tmp'),
    '--region=' + _gcp_region,
], # CULPRIT LEADING COMMA
```

There is a leading comma that causes this variable to be interpreted as a tuple and not a list. I fixed it this way and it worked. I think the latest github repo doesnt have this problem though..

**Comment by zhitaoli** [comment]
I believe the issue was fixed in cab43b36e9b902af1942455f2eb02283bac3f8a6.
